src/taskpane/taskpane.js

```diff
@@ -1224,11 +1316,13 @@ function errorPopUp(type, err) {
     console.log(err);
   } else {
     msOffice.error(type, err);
+    artifactUpdateUI(4);
     console.error("SpiraPlan Import/Export Plugin encountered an error:", err.status ? err.status : "", err.response ? err.response.text : "", err.description ? err.description : "")
     console.info("SpiraPlan Import/Export Plugin: full error is... ", err)
   }
   hideLoadingSpinner();
 }
+artifactUpdateUI(4);
```

Comment:
Why is this line needed? It is outside of the if block so will always run - so it will run once if Google/Excel then a second time outside of the if block. Think it should be deleted, or clearly explained why it needs to runs twice

Reply:
It doesn't need to run twice - that was a mistake. Already fixed.